src/SumoCoders/FrameworkCoreBundle/Resources/assets/js/Framework/Form.js

```diff
@@ -66,141 +66,136 @@ const dateFieldOptions = {
+
+      $(el).datepicker({
+        altField: '#' + $dateWrapper.data('linkField'),
+        altFormat: $dateWrapper.data('linkFormat').replace('yyyy', 'yy'),
+        defaultDate: initialDate
+      })
 
       switch ($dateWrapper.data('dateType')) {
         case 'start':
-          let startDate = this.parseDate($dateWrapper, 'minimumDate');
-          $(el).datepicker('option', 'minDate', startDate);
+          let startDate = this.parseDate($dateWrapper, 'minimumDate')
+          $(el).datepicker('option', 'minDate', startDate)
 
           if ((initialDate !== '') && (initialDate < startDate)) {
-            initialDate = startDate;
+            initialDate = startDate
           }
-          break;
+          break
 
         case 'until':
-          let endDate = this.parseDate($dateWrapper, 'maximumDate');
-          $(el).datepicker('option', 'maxDate', endDate);
+          let endDate = this.parseDate($dateWrapper, 'maximumDate')
+          $(el).datepicker('option', 'maxDate', endDate)
 
           if ((initialDate !== '') && (startDate > initialDate)) {
-            initialDate = startDate;
+            initialDate = startDate
           }
-          break;
+          break
 
         case 'range':
-          startDate = this.parseDate($dateWrapper, 'minimumDate');
-          $(el).datepicker('option', 'minDate', startDate);
+          startDate = this.parseDate($dateWrapper, 'minimumDate')
+          $(el).datepicker('option', 'minDate', startDate)
 
           if ((initialDate !== '') && (initialDate < startDate)) {
-            initialDate = startDate;
+            initialDate = startDate
           }
 
-          endDate = this.parseDate($dateWrapper, 'maximumDate');
-          $(el).datepicker('option', 'maxDate', endDate);
+          endDate = this.parseDate($dateWrapper, 'maximumDate')
+          $(el).datepicker('option', 'maxDate', endDate)
 
           if ((initialDate !== '') && (startDate > initialDate)) {
-            initialDate = startDate;
+            initialDate = startDate
           }
-          break;
+          break
       }
 
       // show initial date if provided
       if (initialDate !== '') {
-        return $(el).val($.datepicker.formatDate(this._dateFieldOptions.dateFormat, initialDate));
+        return $(el).val($.datepicker.formatDate(this._dateFieldOptions.dateFormat, initialDate))
       }
-    });
+    })
 
     $('[data-date-type] a', this.form).on('click', () => {
-      let el = $(this).parents('[data-provider="datepicker"]').find('input:first');
+      let el = $(this).parents('[data-provider="datepicker"]').find('input:first')
       if (!$(el).datepicker('widget').is(':visible')) {
         return $(el).datepicker('show')
       }
-    });
+    })
   }
 
   /* fixes */
-  fixPlaceholders()
-  {
-    /*detect if placeholder-attributes is supported*/
-    jQuery.support.placeholder = (Array.from(document.createElement('input')).includes('placeholder'));
+  fixPlaceholders () {
```

Comment:
Not related to the PR: but is this still relevant. All supported browsers support this, so no need to "fake" this behaviour with Javascript
See: http://caniuse.com/#feat=input-placeholder